src/style.css

```diff
@@ -1 +1,48 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin:;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(1),
+.stars--3 :nth-child(1),
+.stars--4 :nth-child(1),
+.stars--5 :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(2),
+.stars--3 :nth-child(2),
+.stars--4 :nth-child(2),
+.stars--5 :nth-child(2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(3),
+.stars--4 :nth-child(3),
+.stars--5 :nth-child(3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(4),
+.stars--5 :nth-child(4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(5) {
+  background-image: url(./images/star-active.svg);
```

Comment:
You have multiple selectors modifying the background-image property with the same value. Consider combining them to reduce repetition and improve maintainability. For example, you could use a single selector such as '.stars--1 .stars__star, .stars--2 .stars__star, ...' with the appropriate child selector if all stars in a given rating should be active.